CMSgov/design-system
# [WNMGDS-665] Add polyfills required for ie11 support

### Summary
- Update Month Picker Component to add `.includes` property polyfill for IE11 support. 
- Update ChoiceList Component to use `indexOf` instead of `.includes` for IE11 support. 
- Update currency Masked field to use `indexOf` instead of `startsWith`
- Move IE11 support for Tooltips Popper to the Tooltips component instead of the polyfills in `createWebpackConfig`.  This way the applications using the CMSDS don't have do any extra polyfill config

### How to test
- Run the design system doc site locally
- Ensure Month Picker component works correctly on IE11
- Ensure ChoiceList `onComponentBlur` function works on IE11 and other browsers (currently, there're other warning messages showing up on IE11 which we'll look at separately)
- Ensure Masked field for currency works
- Ensure Tooltips component works
